NAMESPACE

```diff
@@ -141,7 +140,7 @@ export(articulation.points)
 export(articulation_points)
 export(as.directed)
 export(as.igraph)
-export(as.phylo)
+export(as.phylo.communities)
```

Comment:
This needs to go to.

Reply:
- Remove the appropriate `#' @export` tag
- Ensure that `as.phylo.communities()` is registered on load (if ape is loaded) or as soon as ape is loaded, by calling `s3_register()` in `.onLoad()` or via `rlang::run_on_load()`